build_baselines.py

```diff
@@ -145,7 +145,7 @@ def parse_iso(file_name):
     }
 
 
-def build_base(iso, md5, replace_existing, vmServer=None, prependString = "", index = "1"):
+def build_base(iso, md5, factory_image, replace_existing, vmServer=None, prependString = "", index = "1"):
```

Comment:
When enhancing options on a method especially optional methods it is best to add to the end of the signature and set a default when possible.

```suggestion
def build_base(iso, md5, replace_existing, vmServer=None, prependString = "", index = "1", factory_image = False):
```

Reply:
Initially, that is what I did. Python required arguments of different types to be grouped together, being the reason why it is placed there for `baseline_builders.py` and `linux_baseline_builders.py`. Additionally, if the default is a desired outcome, we should change `replace_existing` with `replace_existing = False`.

Reply:
The only grouping requirement should be `positional/required` vs `optional/named` arguments as grouped. If this is what you are referring to my suggestions takes this into account by setting a default.

`replace_existing` could be enhanced to be a `optional/named` argument however when it was added there were no optional arguments. Since changing the semantics of that argument is not a goal of this PR I see no reason to add unnecessary changes.

When making changes to an existing method, a risk averse approach by making minimal backwards compatible signature changes is preferred.